pyserini/trectools/_base.py

```diff
@@ -14,9 +14,25 @@
 # limitations under the License.
 #
 
+from __future__ import annotations
+
 from copy import deepcopy
+from enum import Enum
 import pandas as pd
-from typing import Set
+from pyserini.pyclass import autoclass
+from typing import Dict, List, Set
+
+# Wrappers around Anserini classes
+JSimpleSearcherResult = autoclass('io.anserini.search.SimpleSearcher$Result')
```

Comment:
We've created this in search already; just import, otherwise you'll get weird variable shadowing effects...